cli/compat/mod.rs

```diff
@@ -140,3 +140,16 @@ pub(crate) fn add_global_require(
 fn escape_for_single_quote_string(text: &str) -> String {
   text.replace(r"\", r"\\").replace("'", r"\'")
 }
+
+pub fn setup_builtin_modules(
+  js_runtime: &mut JsRuntime,
+) -> Result<(), AnyError> {
+  for module in SUPPORTED_MODULES {
+    // skipping the modules that contains '/' as they are not available in NodeJS repl as well
+    if !module.contains('/') {
+      let script = format!("const {} = require('{}');", module, module);
+      js_runtime.execute_script("setup_builins.js", &script)?;
+    }
+  }
```

Comment:
Let's rewrite this for loop to create a single script and execute it only once. Also I suggest to rename the script to `setup_node_builtins.js`

Reply:
updated and squashed to single commit